assembl/models/langstrings.py

```diff
@@ -464,6 +464,49 @@ def EMPTY(cls, db=None):
     def reset_cache(cls):
         pass
 
+    # Which object owns this?
+    owner_object = None
+    _owning_relns = []
```

Comment:
This becomes a `self._owning_relns` here in Python on the object yes? Not a class variable, if I understand correctly?